cogs/edit_message.py

```diff
@@ -113,7 +113,7 @@ async def edit_message(self, ctx: TeXBotApplicationContext, str_channel_id: str,
         if not channel:
             await self.command_send_error(
                 ctx,
-                message=f"Text channel with ID \"{channel_id}\" does not exist.",
+                message=f'Text channel with ID "{channel_id}" does not exist.',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes